src/components/ReviewList.jsx

```diff
@@ -1,27 +1,29 @@
-import { useEffect } from "react";
+import { useState, useEffect } from "react";
 import { getAllReviews } from "../utils/utils";
 import ReviewCard from "./ReviewCard";
 
 
-
-function ReviewList({reviewList, setReviewList}) {
+function ReviewList({filter}) {
+  const [reviewList, setReviewList] = useState([]);
 
   useEffect(() => {
-    getAllReviews().then((review) => {
+    getAllReviews(filter).then((review) => {
       setReviewList(review)
     });
   }, []);
```

Comment:
No loading state to let users know the reviews are loading. Would be great if you can add one :)